client/src/components/StarsInfoPlayer.tsx

```diff
@@ -0,0 +1,46 @@
+import filledStar from "../assets/FilledStar.svg";
+import unfilledStar from "../assets/UnfilledStar.svg";
+
+type StarsInfoPlayerProps = {
+  playerName: string;
+  score: number;
+};
+
+const StarsInfoPlayer = ({ playerName, score }: StarsInfoPlayerProps) => {
```

Comment:
What's the difference between the `StarsInfoPlayer` and `StarsInfoOpponent`?

If the only difference is colour we can create a variable called `isOpponent` which is a boolean and change the colour dynamically based on that. Let me know if you don't know what I mean.

To make the CSS dynamic you can use a template string. For example:

```javascript
<div className={"fixed bottom-5 ..." + ` bg-${isOpponent ? "[#45BD67]" : "[#FFFFFF]"`}>
```